src/model/commands/oob.rs

```diff
@@ -9,6 +9,41 @@ pub struct Command {
     pub payload: Vec<u8>,
 }
 
+/// # Example
+/// ```
+/// use iroha::model::commands::oob::Command;
```

Comment:
Can we change OOB to ISI? Iroha Special Instructions. It is because Iroha has an instruction set that can be executed for high performance.

Reply:
Yes, of course